source/_posts/angular-youtube.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Youtube Data API
+date: 2020-05-21 16:36:18
+author: Abhishek Rana
+category: Web
+tags:
```

Comment:
Tag should be 3-4 and atleast one must be long tail keyword, which is 2 to 3 word keyword

Reply:
Resolved